### PR TITLE
adding ceviche

### PR DESCRIPTION
**Pull request by TiffanyClark**

### CODE REVIEW

File: src/recipes/ceviche-with-shrimp.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: "Ceviche With Shrimp"
+---
+
+# Ceviche With Shrimp
+
+Ceviche is one of my favorite dishes that I enjoyed when I started my no red meat diet. I first tried Ceviche when my bestfriend brought it to work for Lunch. I loved it so much that I learned how to make it on my own.
+
+## Ingredients
+
+1 bag of cooked Shrimp
```
Comment (jlooper) [inline src/recipes/ceviche-with-shrimp.md:12]: please make these and other lists bullet points to format well: 

- ingredient 1
- ingredient 2

File: src/recipes/ceviche-with-shrimp.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Tools
+
+Bowl
+juicer
+list of tools
+
+## Method
+
+Cut up the tomatoe, purple onion, jalpeno pepper, cilantro
```
Comment (jlooper) [inline src/recipes/ceviche-with-shrimp.md:29]: Tomato (please check spelling)

File: src/recipes/ceviche-with-shrimp.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Add Ingredients to Bowl
+Peel shrimp tail off.
+Cut each lime in half and squeeze juice into bowl
+Mix all the ingredients together
+Let ceviche sit in fridge to chill
+
+## Tips
+
+To add a better taste, I also add lemon peper and garlic powder to add seasoning. you can add any kind of totilla chips to the dish.
```
Comment (jlooper) [inline src/recipes/ceviche-with-shrimp.md:38]: looks yummy! you might add this either to the dinner category or let it stay in 'everything else'

**Review by jlooper — CHANGES_REQUESTED** [review]
just a few edits! I love ceviche

**Comment by TiffanyClark** [comment]
I made the correct changes for recipe

**Review by jlooper — APPROVED** [review]
going to merge this but edit a few things

**Pull request merged**